module.txt

```diff
@@ -1,11 +1,11 @@
 {
     "id" : "StructureTemplates",
-    "version" : "0.3.0-SNAPSHOT",
+    "version" : "0.3.1-SNAPSHOT",
     "author" : "Florian",
     "displayName" : "Structure Templates",
     "description" : "A module that adds 'structure templates' which can be used to spawn whole structures for faster building.",
     "dependencies" : [
-        {"id" : "Core", "minVersion" : "2.0.0"}
+        {"id" : "Core", "minVersion" : "2.2.1"}
```

Comment:
Slight doubt here: While this makes the dependency match the current version is the newer min version really _required_ ?

In pretty much any case right now it wouldn't matter, but I am wondering if as a practice we should avoid bumping minimums unless there's a specific reason. Which there may be here, unsure :-)

Reply:
I am not sure if any older versions of Core would *conflict* with CoreAssets and CoreBlocks, exactly; they're not code, after all. It may still be possible to have an older Core alongside the new modules and have no side effects beyond messy duplication.

Reply:
Where does the dependency on `CoreBlocks` come from? If it is a transitive/indirect dependencycom from _Core_ the version bump is likely required here. 
If we add an explicit dependency on _CoreBlocks_ in this module then we might be able to go without bumping minimal version for _Core_?

@Qwertygiy @Cervator what do you think? I'd like to merge this, but rather _with the explicit dependency on _CoreBlocks_ in `module.txt`.

Reply:
Agreed, that is how I proceeded with the rest of the modules.